README.md

```diff
@@ -11,6 +11,7 @@ on:
 
 jobs:
   Documenter:
+    permissions: write-all
```

Comment:
This is not a good practice. Instead, only the minimal permissions that are needed to make the workflow work should be given to the token.

Reply:
Well spotted :+1: Based on https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions?ref=#permissions, the permissions are now set to
```
permissions:
    contents: write
```
I've tested this in a repository of mine (https://github.com/rikhuijzer/DiscountedCashFlows.jl/commits/main).